cv_next/server/api/cvs.ts

```diff
@@ -4,7 +4,7 @@ import MyLogger from "@/server/base/logger";
 import Categories from "@/types/models/categories";
 import Definitions from "../../lib/definitions";
 import SupabaseHelper from "./supabaseHelper";
-import { PostgrestError } from "@supabase/supabase-js";
+import { PostgrestError, } from "@supabase/supabase-js";
 import { filterValues } from "@/app/feed/components/filterPanel";
 
 export const revalidate = Definitions.CVS_REVALIDATE_TIME_IN_SECONDS;
```

Comment:
REVALID_DATE, global  const needs to be capitel with _ between words

Reply:
This isn't used so can be deleted
and revalidate is one word